.github/workflows/deploy_staging.yml

```diff
@@ -0,0 +1,91 @@
+name: Deploy Pandora Staging
+
+on:
+    # Run this workflow on push to master (incl PR merges)
+    push:
+        branches: [ master ]
+
+    # Allow running this workflow manually from the Actions tab
+    workflow_dispatch:
+
+jobs:
+  deploy_staging_branch:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: azure/login@v1
+      with:
+          creds: ${{ secrets.AZURE_STATIC_SITES }}
+
+    - name: Node.js build
+      id: build
+      uses: actions/setup-node@v1
+      with:
+        node-version: '14.x'
+    - run: npm install
+    - run: npm run build
+
+    - name: Upload to blob storage
+      uses: azure/CLI@v1
+      with:
+        creds: ${{ secrets.AZURE_STATIC_SITES }}
+        inlineScript: |
+            az storage blob upload-batch \
+              --account-name zooniversestatic \
+              --content-cache-control 'public, immutable, max-age=604800' \
+              --destination '$web/pandora.zooniverse.org' \
+              --source ./build
+            az storage blob upload \
+              --account-name zooniversestatic \
+              --content-cache-control 'public, max-age=60, s-maxage=60' \
+              --container-name '$web' \
+              --name 'pandora.zooniverse.org/index.html' \
+              --file ./build/index.html
+
+    - name: Slack notification
+      uses: 8398a7/action-slack@v3
+      if: always()
+      env:
+        SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+      with:
+        fields: took
+        status: custom
+        custom_payload: |
+          {
+            "channel": "#deploys",
+            "icon_emoji": ":octocat:",
+            "username": "Deploy Action",
+            "attachments": [{
+              "color": '${{ job.status }}' === 'success' ? 'good' : '${{ job.status }}' === 'failure' ? 'danger' : 'warning',
+              "mrkdwn_in": ["text"],
+              "author_name": "${{ github.actor }}",
+              "author_link": "https://github.com/${{ github.actor }}/",
+              "author_icon": "https://github.com/${{ github.actor }}.png?size=40",
+              "title": "Pandora Staging deploy complete",
+              "title_link": "https://pandora.preview.zooniverse.org",
```

Comment:
```suggestion
              "title_link": "https://pandora.zooniverse.org",
```

Reply:
`pandora.` is the staging domain, `translations.` is production.

Reply:
The original npm script I see deploying staging to `pandora.zooniverse.org`. Is it now `pandora.preview.zooniverse.org` on Azure?

Reply:
Oh, I read this backwards. Yeah you're right